manifests/base_apps.pp

```diff
@@ -93,12 +93,19 @@
 
         # portreserve will only start if there is a file in the conf directory
         # such as: cups ipp dhcpd named slapd ldaps
+        file { '/etc/portreserve/sshd':
```

Comment:
There are some workarounds present in spec/acceptance/suites/default/00_simp_spec.rb:68 that could maybe be removed